index.html

```diff
@@ -4,6 +4,19 @@
 		<meta charset="utf-8">
 		<meta http-equiv="X-UA-Compatible" content="IE=edge">
 		<title>Simulateur de coût d'embauche</title>
+		<style>
+			body {
+				font-family: 'Helvetica Neue', Arial, Helvetica, sans-serif;
+				line-height: 2.5;
+			}
+
+			/* Customize the widget's style */
+			body .SGMAPembauche {
```

Comment:
Is there any reason for increasing specificity?
We want to make as simple as possible for clients to customise our widget’s external appearance, yet try to keep control of its inner properties (while still allowing overrides if needed).
If we default to increased specificity, we introduce some uncertainty (“how will my styles be treated if I don't have the `body` selector here?”), and make sure the client's styles will always override ours (at least in the current implementation).